bytecomp/bytelink.ml

```diff
@@ -623,23 +626,21 @@ let build_custom_runtime prim_name exec_name =
     if not !Clflags.with_runtime
     then ""
     else "-lcamlrun" ^ !Clflags.runtime_variant in
-  let debug_prefix_map =
-    if Config.c_has_debug_prefix_map && not !Clflags.keep_camlprimc_file then
-      let flag =
-        [Printf.sprintf "-fdebug-prefix-map=%s=camlprim.c" prim_name]
-      in
-        if Ccomp.linker_is_flexlink then
```

Comment:
I believe this was the only use of `Ccomp.linker_is_flexlink`

Reply:
Yes, indeed it is! Another victory - no longer needing a function to work out if the C compiler pretending to be a linker is in fact a linker pretending to be a C compiler 🤯